src/backend/distributed/citus--5.1-3--5.1-4.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION worker_apply_shard_ddl_command(bigint, text, text)
```

Comment:
This creates the new function in the public schema on a new installation. Could you create it inside pg_catalog;
pg_catalog.worker_apply_shard_ddl_command